examples/images_contours_and_fields/multi_image.py

```diff
@@ -4,80 +4,46 @@
 ===========
 
 Make a set of images with a single colormap, norm, and colorbar.
-
-It also illustrates colorbar tick labelling with a multiplier.
 """
 
-from matplotlib.pyplot import figure, show, sca, sci
-from matplotlib import cm, colors
-from matplotlib.font_manager import FontProperties
+from matplotlib import colors, pyplot as plt
 import numpy as np
 
+np.random.seed(19680801)
```

Comment:
there's been chatter on the numpy mailing this about why it's better to use `RandomState`. I won't purport to fully understand it, but it might be worth getting clarity on that and updating our examples to include their best practices at some point.

Reply:
It's very clear that RandomState is better for isolation, but this should be changed across the entire docs at once rather than in a piecemeal fashion to keep things consistent.

Reply:
Agreed. That's what I was trying to say.